TESTS/mbed_hal_fpga_ci_test_shield/basic/main.cpp

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
The current location of this file (`TESTS/mbed_hal_fpga_ci_test_shield/basic/main.cpp`) is a bit misleading. I think this directory should be renamed to something like `rtos-basic`.